cp1616/config/test_io_controller_config.yaml

```diff
@@ -0,0 +1,43 @@
+#Data comming from PROFINET network
+input:
```

Comment:
How did you come up with the structure of this yaml?  It seems the `module_x` are not required.  Could this be a simple flat list of data structures?